docs/cashmere-import.scss

```diff
@@ -1,4 +1,2 @@
-//this simplifies importing SASS files from the Cashmere library
-@import "../lib/src/colors.scss";
-@import "../lib/src/functions.scss";
-@import "../lib/src/tables.scss";
+
```

Comment:
If we imported the table styles here we might be able to avoid the really long path in table-demo.component.ts. Although I can see why we don't necessarily always want to import everything.

Reply:
The problem I was having with tables was that it imports the other common .scss files as well and I couldn't find paths that everything was happy with.  In desperation I decided to separate it out. :(